### DISCUSSION TITLE
Feature request -  slides are no longer in the html but apart, unitary and included (like a php include ())

### DISCUSSION THREAD

**Discussion by heiwanasenshi** [hakimel/reveal.js#3064] [Ideas]
I don't know if it's possible because I'm not a developer.
I would find it nice that the slides are no longer in the html but apart, unitary and included (like a php include ()) dans le fichier html. This allows modifications in the slides by modifying only the file corresponding to the slide without modifying the html code and avoids errors. It also makes it smoother from my point of view
I think this makes reveal.js even more accessible to non-developers.

**Comment by datavectors** [comment]
This reply might be too late for the OP. 
But I had similar requirement to modularise the build of slides.
The optimum solution for me was to use just after `<div "slides">`: 

`php require "slide-01.html";`
`php require "slide-02.html";`
`php require "slide-nn.html";`

to insert each slide section in a list of "require" statements. 
By using php I can then apply php vars to the slides.
Other methods I looked at depend on whether Windows or Linux.
Basically you breakup distributed index.html into three parts.

start.html
slides.html
end.html

And then use either cat in command line to concatenate files, or python to join the files.
Markdown @import is another I tried.
